### ISSUE TITLE
Create a landing page for the website.

### ISSUE THREAD

**Issue by GauriKashyap** [GauriKashyap/Portfolio.react#2]
In order to make this website cool we need to add a landing page/home page for our website which contains basic intro about ourselves and maybe what all the website is all about.

This issue is open for for all and can have PRs from multiple people as this template is based completely on UI/UX.

**Comment by KarthikS373** [comment]
Hey can I please work upon this issue, if it is available?

**Comment by GauriKashyap** [comment]
Yes you may start making your PRs, landing page is more of frontend hence it's open for all. UI which are influential will be merged.

Anyone can contribute in this issue, it's open for all.

**Comment by KarthikS373** [comment]
@GauriKashyap is it okay to revamp the project structure a bit
```
 - src
   - components
   - pages
   - hooks
   - utils
   - assets
```

**Comment by GauriKashyap** [comment]
Yes until and unless there are no conflicts.

**Comment by GauriKashyap** [comment]
@KarthikS373 this has been assigned to you, as not much people showed interest.

**Comment by KarthikS373** [comment]
yup working on it...

**Comment by GauriKashyap** [comment]
@KarthikS373 what's the status on the landin page?

**Comment by KarthikS373** [comment]
It's almost ready. Will make a few more optimisation and UI fixes and will PR within a few hours
